Kernel/Bus/USB/Drivers/HID/MouseDriver.cpp

```diff
@@ -67,7 +67,7 @@ ErrorOr<void> MouseDriver::initialize_device(USB::Device& device, USBInterface c
         USB_REQUEST_SET_CONFIGURATION, configuration.configuration_id(), 0, 0, nullptr));
 
     auto const& endpoint_descriptor = interface.endpoints()[0];
-    auto interrupt_in_pipe = TRY(USB::InterruptInPipe::create(device.controller(), device, endpoint_descriptor.endpoint_address, endpoint_descriptor.max_packet_size, 10));
+    auto interrupt_in_pipe = TRY(USB::InterruptInPipe::create(device.controller(), device, endpoint_descriptor.endpoint_address & 0xf, endpoint_descriptor.max_packet_size, 10));
```

Comment:
should USB::InterruptInPipe::create verify that the top nibble is 0? or can the top 4 bits be set when that's called from other places?

Reply:
I added a check in the USB::InterruptInPipe base class, USB::Pipe.